android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -157,13 +158,25 @@ public void sendToNotificationCentre(Bundle bundle) {
                 title = context.getPackageManager().getApplicationLabel(appInfo).toString();
             }
 
-            NotificationCompat.Builder notification = new NotificationCompat.Builder(context)
+            String channelId = "step_reminders_channel_id";
+            NotificationManager notificationManager = notificationManager();
+
+            NotificationCompat.Builder notification = new NotificationCompat.Builder(context, channelId)
+                    .setChannelId(channelId)
```

Comment:
I'm guessing `setChannelId()` isn't necessary since you define the channelId when creating the Notification Builder.